### ISSUE TITLE
Icon Request: icon-fruit-apple (apple fruit)

### ISSUE THREAD

**Issue by rpowers4** [FortAwesome/Font-Awesome#2979]
Hello!

Just started using font-awesome and love the icons. Was hoping that you could create an icon for food (potentially an apple...but not the Apple Inc logo w/ the bite taken out). This would be helpful (at the very least) for users developing apps for State Agencies where you need an icon for representing "Food Assistance (SNAP)" benefits. 

Attached a color example: 

![apple](https://f.cloud.github.com/assets/6809101/2287502/c99c7cdc-9fed-11e3-84e9-fdfdc675464f.png)

Thanks!

Bobby

**Comment by sirenAri** [comment]
+1
I would love to have an apple (fruit) icon.  I work for a school district and I'm using an apple icon to indicate a menu for the faculty and staff (apple for the teacher, right?).

If it helps, I made a vector version of the apple example in the first post.  I also have a Photoshop PSD document with it as well if you'd like.  I could also easily enough convert it to any format you'd like.
![apple](https://cloud.githubusercontent.com/assets/10656735/5859872/2d7adcfc-a223-11e4-984d-031726cacbfe.png)

Thanks for considering this!

**Comment by tagliala** [comment]
Adding to 5.0.0 milestone via #1895 

Note that this could be shifted to a future release

**Comment by G-Driver** [comment]
+1

**Comment by vita10gy** [comment]
+1

**Comment by Crissov** [comment]
- 🍏 [U+1F34F](http://emojipedia.org/green-apple/) Unicode emoji #222 
- 🍎 [U+1F34E](http://emojipedia.org/red-apple/)

**Comment by jrowellfx** [comment]
Also adding a +1 to this request (I inadvertently opened another similar request-it was closed so I could add to this thread).

I found an excellent reference for what I think the simplicity of the symbols should be - here's a link.

https://s-media-cache-ak0.pinimg.com/originals/5c/90/f3/5c90f3bcfebab01b94bd1bd8017aa42d.jpg

apple-fruit, banana-fruit, cherries-fruit, ... and some others could be useful.  The above link has other related fruit that might also be good to have as a family of symbols.

Thanks so much for considering this request!
